src/me/EtienneDx/RealEstate/Config.java

```diff
@@ -212,4 +220,105 @@ public void loadConfig()
+
+        HashMap<String, CustomizableMessage> defaults = new HashMap<String, CustomizableMessage>();
+        // initialize defaults
+        this.addDefault(defaults, Messages.NoTransactionFound, "No transaction found at your location!", null);
+        this.addDefault(defaults, Messages.PageMustBePositive, "Page must be a positive option!", null);
+        this.addDefault(defaults, Messages.PageNotExists, "This page does not exist!", null);
+        this.addDefault(defaults, Messages.RenewRentNow, "Automatic renew is now $a{0} $bfor this {1}", "0: the status; 1: a claim type");
+        this.addDefault(defaults, Messages.RenewRentCurrently, "Automatic renew is currently $a{0} $bfor this {1}", "0: the status; 1: a claim type");
+
+
+        // load the config file
+        FileConfiguration config = YamlConfiguration.loadConfiguration(new File(messagesFilePath));
+
+        // for each message ID
+        for (Messages messageID : messageIDs) {
+            //get default for this message
+            CustomizableMessage messageData = defaults.get(messageID.name());
+
+            // if default is missing, log an error and use some fake data for now so that the plugin can run
+            if (messageData == null) {
+                RealEstate.instance.log.info("Missing message for " + messageID.name() + ".  Please contact the developer.");
+                messageData = new CustomizableMessage(messageID, "Missing message!  ID: " + messageID.name() + ".  Please contact a server admin.", null);
+            }
+
+            // read the message from the file, use default if necessary
+            this.messages[messageID.ordinal()] = config.getString("Messages." + messageID.name() + ".Text", messageData.text);
+            config.set("Messages." + messageID.name() + ".Text", this.messages[messageID.ordinal()]);
+
+            this.messages[messageID.ordinal()] = this.messages[messageID.ordinal()].replace('$', (char) 0x00A7);
+
+            if (messageData.notes != null) {
+                messageData.notes = config.getString("Messages." + messageID.name() + ".Notes", messageData.notes);
+                config.set("Messages." + messageID.name() + ".Notes", messageData.notes);
+            }
+        }
+
+        //save any changes
+        try {
+            config.options().header("Use a YAML editor like NotepadPlusPlus to edit this file.  \nAfter editing, back up your changes before reloading the server in case you made a syntax error.  \nUse dollar signs ($) for formatting codes, which are documented here: http://minecraft.gamepedia.com/Formatting_codes");
+            config.save(messagesFilePath);
+        } catch (IOException exception) {
+            RealEstate.instance.log.info("Unable to write to the configuration file at \"" + messagesFilePath + "\"");
+        }
+
+        defaults.clear();
+        RealEstate.instance.log.info("Customizable messages loaded.");
+        System.gc();

```

Comment:
That's considered a [bad practice](https://www.baeldung.com/java-system-gc), java does this by itself when needed

```suggestion

```